cmd/firenear/cli/logging.go

```diff
@@ -0,0 +1,5 @@
+package cli
+
+import "github.com/streamingfast/logging"
+
+var zlog, _ = logging.RootLogger("firenear", "github.com/streamingfast/firehose-near/cmd/firenear")
```

Comment:
Fully qualified name should fit with current package:
```suggestion
var zlog, _ = logging.RootLogger("firenear", "github.com/streamingfast/firehose-near/cmd/firenear/cli")
```